1.9/usage/cli/command-reference/dcos-task/dcos-task-exec.md

```diff
@@ -0,0 +1,34 @@
+---
+post_title: dcos task exec
+menu_order: 0
+---
+
+# Description
+Launch a process inside of a task container.
+
+# Usage
+
+```bash
+dcos task exec [OPTION]
+```
+
+# Options
+
+| Name, shorthand | Default | Description |
+|---------|-------------|-------------|
+| `--interactive, -i`   |             |  Attach a STDIN stream to the remote command for an interactive session. |
+| `--tty, -t`   |             |  Attach a TTY to the remote stream. |
+
+# Positional arguments
+
+| Name, shorthand | Default | Description |
+|---------|-------------|-------------|
+| `<args>`   |             |  Additional arguments to pass to the command. |
```

Comment:
I think it's more logical for required arguments to appear in this list in the order they would be written on the command line, but I'm fine either way.